drivers/volume/kdmp/kdmp.go

```diff
@@ -778,6 +797,11 @@ func (k *kdmp) StartRestore(
 			msg := fmt.Sprintf("unable to find backup uid from applicationbackup %s/%s", restore.Namespace, restore.Spec.BackupName)
 			return nil, fmt.Errorf(msg)
 		}
+		getUIDFromApp := false
+		podUserId, podGroupId, psaIsEnabled, err := utils.GetPsaEnabledAppUID("", restoreNamespace, backup, getUIDFromApp)
+		if err != nil {
+			logrus.Errorf("failed to get the UID and GID for pvc %s %v", pvc.Name, err)
```

Comment:
On error, we should return from the reconciler and not proceed further

Reply:
+1